C/tests/c4DatabaseTest.cc

```diff
@@ -726,22 +728,31 @@ N_WAY_TEST_CASE_METHOD(C4DatabaseTest, "Test delete while database open", "[Data
     C4Database* otherConnection = c4db_openAgain(db, &err);
     REQUIRE(otherConnection);
     c4db_close(otherConnection, &err);
-
+#ifdef FAIL_FAST
     auto start = chrono::system_clock::now();
+#endif
     {
         ExpectingExceptions e;
         CHECK(!c4db_delete(otherConnection, &err));
     }
-
+#ifdef FAIL_FAST
     auto end = chrono::system_clock::now();
+#endif
     c4db_release(otherConnection);
 
+#ifdef FAIL_FAST
     auto timeTaken = chrono::duration_cast<chrono::seconds>(end - start);
     CHECK(timeTaken < 2s);
+#endif
     CHECK(err.code == kC4ErrorBusy);
 
-    auto message = slice(c4error_getDescription(err));
-    CHECK(message == "Can't delete db file while the caller has open connections");
+    C4SliceResult message = c4error_getDescription(err);
+#ifdef FAIL_FAST
+    CHECK(slice(message) == "LiteCore error 16 \"Can't delete db file while the caller has open connections\"");
```

Comment:
I remember the main goal to fail fast is to prevent the 3rd party developer to open a database with the same file?
Not sure why you print `Can't delete db file while the caller has open connections` It doesn't look right to me.

Reply:
It's fine to open the same db file multiple times. The goal is to fail immediately if a developer tries to _delete_ a database that's still open in another connection. But even without that, we'll still fail, but after a delay.

Reply:
Jim came up with that message. He wrote the test case. I think "caller" here means the app developer. Connections mean database handles that the app obtains from calling c4db_open().